static/app/views/settings/projectDebugFiles/externalSources/customRepositories/status.tsx

```diff
@@ -23,9 +24,9 @@ function Status({details, onEditRepository}: Props) {
     return <Placeholder height="14px" />;
   }
 
-  const {pendingDownloads, appstoreCredentialsValid, lastCheckedBuilds} = details ?? {};
+  const {pendingDownloads, credentials, lastCheckedBuilds} = details ?? {};
 
-  if (appstoreCredentialsValid === false) {
+  if (areAppStoreConnectCredentialsValid(credentials) === false) {
```

Comment:
```suggestion
  if (!areAppStoreConnectCredentialsValid(credentials)) {
```

Reply:
should we maybe just 

```suggestion
  if (credentials?.status !== 'valid') {
```